modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/Framework.java

```diff
@@ -69,6 +71,7 @@ public class Framework implements IFramework, IShuttableFramework {
     private IAuthStore                         authStore;
     private RBACService                        rbacService;
     private IStreamsService                    streamsService;
+    private CacheUsers                         usersCache;
```

Comment:
Is this needed? It looks like it's not doing anything

Reply:
Great spot, thanks - removed in https://github.com/galasa-dev/galasa/pull/468/commits/019cb96c7d09d031ed4a92ca63674dcc1fd13a29